gallery/Cohorts-microsoft.insights-components.json

```diff
@@ -0,0 +1,78 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
+	"version": "TemplateGallery/1.0",
+	"categories": [
+		{
+			"id": "AuthenticatedUsers",
+			"name": "Authenticated Users",
+			"templates": [
+				{
+					"id": "Cohorts/AuthenticatedUsers/Engaged Authenticated Users",
+					"author": "Microsoft",
+					"description": "description goes here",
```

Comment:
>"description": "description goes here", [](start = 5, length = 39)

for these cohorts ones make the desc empty string or the same as name?  don't want "description goes here" translated literally :D